org.es_de.emulationstation-de.yml

```diff
@@ -0,0 +1,87 @@
+#  SPDX-License-Identifier: MIT
+#
+#  EmulationStation Desktop Edition
+#  org.es_de.emulationstation-de.yml
+#
+#  Flatpak manifest file for use with flatpak-builder.
+#
+
+app-id: org.es_de.emulationstation-de
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+command: emulationstation
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --filesystem=host:ro
+  - --filesystem=/var/lib/flatpak/exports/bin:ro
+  - --filesystem=home
+  - --filesystem=~/.var/app
+  - --share=network
+  - --talk-name=org.freedesktop.Flatpak
+  - --env=PATH=/app/bin:/var/lib/flatpak/exports/bin:/usr/bin:/usr/local/bin:/usr/local/sbin:/usr/sbin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/var/lib/snapd/snap/bin
+cleanup:
+  - /include
+  - /share/ffmpeg
+  - /lib/cmake
+  - /lib/debug
+  - /lib/pkgconfig
+  - /lib/*.a
```

Comment:
```suggestion
  - '*.a'
```

Reply:
I did change it, it was the following before:
/lib/libfreeimage.a
And I changed it to:
/lib/*.a
There are no .a files outside the lib directory.

Running this command confirms it:
find ~/.local/share/flatpak/app/org.es_de.emulationstation-de/ -name *.a | wc -l
0

Reply:
*sigh*

This will remove ANY `*.a`. Period. It doesn't matter if they are just in `lib`.

Reply:
Yes I'm aware of how regular expressions work. There are however no .a files outside the /lib/ directory. But I've still made the change and committed it now to satisfy your request.